packages/regenesis-surgery/scripts/constants.ts

```diff
@@ -0,0 +1,80 @@
+export const EOA_CODE_HASHES = [
+  '0xa73df79c90ba2496f3440188807022bed5c7e2e826b596d22bcb4e127378835a',
+  '0xef2ab076db773ffc554c9f287134123439a5228e92f5b3194a28fec0a0afafe3',
+]
+
+export const UNISWAP_V3_FACTORY_ADDRESS =
+  '0x1F98431c8aD98523631AE4a59f267346ea31F984'
+
+export const UNISWAP_V3_NFPM_ADDRESS =
+  '0xC36442b4a4522E871399CD717aBDD847Ab11FE88'
+
+export const UNISWAP_V3_LIB_ADDRESSES = [
+  // Position
+  '0x18F7E3ae7202e93984290e1195810c66e1E276FF',
+  // Oracle
+  '0x17b0f5e5850e7230136df66c5d49497b8c3be0c1',
+  // Tick
+  '0x47405b0d5f88e16701be6dc8ae185fefaa5dca2f',
+  // TickBitmap
+  '0x01d95165c3c730d6b40f55c37e24c7aac73d5e6f',
+  // TickMath
+  '0x308c3e60585ad4eab5b7677be0566fead4cb4746',
+  // SwapMath
+  '0x198dcc7cd919dd33dd72c3f981df653750901d75',
+  // UniswapV3PoolDeployer
+  '0x569e8d536ec2dd5988857147c9fcc7d8a08a7dbc',
+  // NFTDescriptor
+  '0x042f51014b152c2d2fc9b57e36b16bc744065d8c',
+]
+
+export const UNISWAP_V3_CONTRACT_ADDRESSES = [
+  // PoolDeployer
+  '0x569E8D536EC2dD5988857147c9FCC7d8a08a7DBc',
+  // UniswapV3Factory
+  '0x1F98431c8aD98523631AE4a59f267346ea31F984',
+  // ProxyAdmin
+  '0xB753548F6E010e7e680BA186F9Ca1BdAB2E90cf2',
+  // TickLens
+  '0xbfd8137f7d1516D3ea5cA83523914859ec47F573',
+  // Quoter
+  '0xb27308f9F90D607463bb33eA1BeBb41C27CE5AB6',
+  // SwapRouter
+  '0xE592427A0AEce92De3Edee1F18E0157C05861564',
+  // NonfungiblePositionLibrary
+  '0x42B24A95702b9986e82d421cC3568932790A48Ec',
+  // NonfungibleTokenPositionDescriptor
+  '0x91ae842A5Ffd8d12023116943e72A606179294f3',
+  // TransparentUpgradeableProxy
+  '0xEe6A57eC80ea46401049E92587E52f5Ec1c24785',
+  // NonfungibleTokenPositionManager
+  '0xC36442b4a4522E871399CD717aBDD847Ab11FE88',
+  // UniswapInterfaceMulticall
+  '0x1F98415757620B543A52E61c46B32eB19261F984',
```

Comment:
this is the same address as `UniswapV3Factory`; i think on L1 this is a different address but on L2 this is: https://optimistic.etherscan.io/address/0x90f872b3d8f33f305e0250db6a2761b354f7710a#code which i think we can just recompile as is.